docker/docusaurus.conf

```diff
@@ -1,8 +1,8 @@
 server {
-       listen 80;
-       listen [::]:80;
+       listen 80 default_server;
+       listen [::]:80 default_server;
 
-       server_name docs.csquare.run;
+       server_name docs.csquare.run docs.csquare.app;
```

Comment:
In fact, this is not necessary. We are deploying a single backend so there is no problem to not filter the request based on the host. I suggest to remove those lines.